### DISCUSSION TITLE
PSP Launcher: The USB device cannot be accessed

### DISCUSSION THREAD

**Discussion by ENDrain** [aldostools/webMAN-MOD#766] [Q&A] [answered]
Hello, I'm trying to run a PSP iso. It says I need a PSP Launcher, so I'm trying to install those (PSP Remasters and/or PSP minis), and getting this error: "The USB device cannot be accessed".

This happens both when installing from webMAN addons menu or from manually downloaded packages. I've tried, in roughly this order:
- rebuilding database
- reinstalling webMAN
- rebuilding database again
- mounting an ISO and trying then
- running the ~re~unmap command

![photo_2022-08-31_21-06-39](https://user-images.githubusercontent.com/29266283/187748775-ccd0d632-9a12-4dd8-a29b-b8656186c9fd.jpg)

**Comment by aldostools** [comment]
This happen when the PSP launchers are already installed and there is not a PSP ISO mounted.

To unlock the PSP launchers you need to mount a PSP ISO or select webMAN Setup > Redirect USB000 > Unmap PS3

**Reply by ENDrain** [reply]
> This happen when the PSP launchers are already installed and there is not a PSP ISO mounted.
> 
> To unlock the PSP launchers you need to mount a PSP ISO or select webMAN Setup > Redirect USB000 > Unmap PS3

I tried both already

**Reply by aldostools** [reply]
> > This happen when the PSP launchers are already installed and there is not a PSP ISO mounted.
> > To unlock the PSP launchers you need to mount a PSP ISO or select webMAN Setup > Redirect USB000 > Unmap PS3
> 
> I tried both, as I've already said in my post.

Then you did something wrong... Try opening http://0/remap.ps3

That you should all the PSP launchers and should let you reinstall them.

**Comment by ENDrain** [comment, marked as answer]
Ok, I did something and it worked.

I tried installing "Priority OFF for PSP Launcher" package from brewology. This sent my console into drive repairing mode, after which I was able to intsall launchers. But webMAN-MOD went missing, I had to reinstall it. Luckily the settings remained intact. I also ran "Priority ON..." package, just in case. 

Maybe all I had to do was try to reintsall launchers with webMAN uninstalled.

The ISO is not working, but that's another story I guess.
